plugins/org.eclipse.emf.mwe.utils/src/org/eclipse/emf/mwe/utils/Writer.java

```diff
@@ -40,14 +40,21 @@ public void setOPTION_SCHEMA_LOCATION(final boolean option_schema_location) {
 
 	private boolean OPTION_SCHEMA_LOCATION_IMPLEMENTATION = true;
 
-	private final String encoding = null;
+	private String encoding = null;
 
 	private boolean multipleResourcesInCaseOfList = false;
 
 	private boolean cloneSlotContents = false;
 	
 	private boolean ignoreEmptySlot = false;
 
+	/**
+	 * @since 1.6.0
+	 */
+	public void setEncoding(final String encoding) {
+		this.encoding = encoding;
```

Comment:
Should we try to validate the encoding in the `checkConfiguration` method?

Reply:
Makes sense, done that.